libs/tup-components/src/tickets/TicketCreateForm.tsx

```diff
@@ -0,0 +1,227 @@
+import React, { useMemo } from 'react';
+import { Formik, Form } from 'formik';
+import { Link } from 'react-router-dom';
+//import ReCAPTCHA from 'react-google-recaptcha';
+import {
+  UserProfile,
+  useTicketCreate,
+  useTicketCreateNoAuth,
+} from '@tacc/tup-hooks';
+import { FormikInput } from '@tacc/core-wrappers';
+import * as Yup from 'yup';
+import {
+  Alert,
+  Col,
+  Container,
+  FormGroup,
+  ModalBody,
+  ModalFooter,
+  Row,
+} from 'reactstrap';
+import { Button, FileInputDropZoneFormField } from '@tacc/core-components';
+import { formValues } from './';
+import './TicketCreateForm.scss';
```

Comment:
Requests:

1. Rename to `TicketCreateForm.global.css`. [Reference.](https://github.com/TACC/tup-ui/blob/task/tup-ui-styles/apps/tup-ui/src/styles/README.md)
2. Convert SCSS syntax—if any—to native CSS or  [PostCSS syntax](https://github.com/csstools/postcss-plugins/tree/main/plugin-packs/postcss-preset-env#glossary).